src/Endpoints/Traits/ListsResource.php

```diff
@@ -9,11 +9,14 @@
     /**
      * Get all from resource.
      *
+     * @param array $query query parameters of list request
      * @return Response
      */
-    public function all()
+    public function all(array $query=array())
```

Comment:
I prefer using the short array syntax and also it is good to leave space between the = sign - `$query = []`